source/Playnite/GamesEditor.cs

```diff
@@ -325,6 +326,12 @@ void cancelStartup(string message)
                     {  "SelectedRomFile", startingArgs.SelectedRomFile }
                 };
 
+                wasHdrEnabled = HdrUtilities.IsHdrEnabled();

```

Comment:
All calls to `HdrUtilities` should be in try/catch (also logging potential failure). I understand that this should generally fail in crashy way, but I've seen people "customizing" their Windows installs to the point that more trivial things would fail in nonsensical way.

Might be worth handling it directly in HdrUtilities, not sure.

Reply:
Added error logging into `HdrUtilities.cs` and I also added try/catches around the public methods in `HdrUtilities` so that anywhere that calls it doesn't need to do any exception handling. If something weird happens, it would just be treated as if HDR is not supported/enabled